backend/typescript/server.ts

```diff
@@ -0,0 +1,26 @@
+import cors from "cors";
+import express from "express";
+
+import { mongo, sequelize } from "./models/index";
+// import PgPerson from "./models/person.pgmodel";
+// import MgPerson from "./models/person.mgmodel";
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+// app.get("/", async (_req, res) => {
+//     await PgPerson.create({name: "First Last", email: "first@last.com"});
+//     await MgPerson.create({name: "First Last", email: "first@last.com"});
+//     res.send("Created some records!");
+// });
+
```

Comment:
Just a quick way to test the models, keeping it for demo purposes but should be removed soon